### ISSUE TITLE
Fully Incompatible (?) with Adetailer 

### ISSUE THREAD

**Issue by zcatharisis** [v0xie/sd-webui-incantations#21]
I encountered an issue where PAG would apply normally on the first image generated in A1111, but errored out and didn't apply its effects after any subsequent generations. ~~This was solved by adding ",incantations" to Settings>Adetailer>Script names to apply to ADetailer (separated by comma).~~ 

~~It should maybe be added to the readme or pinned somewhere in case another user experiences the same errors.~~

Disregard above, extension is fully incompatible with adetailer enabled. It causes PAG to error out after a few generations. Adetailer needs to be disabled for PAG to function properly.

I'd also like to note that, at least in my setup, PAG is fully compatible with Hypertile, Negpip(https://github.com/hako-mikan/sd-webui-negpip/), and Diffusion CG(https://github.com/Haoming02/sd-webui-diffusion-cg). All these are extensions which (if I understand correctly) affect the denoising process.

Currently on 1.9.0 A1111, all extensions updated to latest version as of posting, running a 1.5 checkpoint, all on a 2060 Super.

**Comment by TheOctonaut** [comment]
Hi @zcatharisis just want to check f I'm having this issue, could you describe the error message you saw?

**Comment by zcatharisis** [comment]
> Hi @zcatharisis just want to check f I'm having this issue, could you describe the error message you saw?

I tried to run adetailer and PAG, and weirdly enough it works now. The only relevant thing I remember doing since last month till now is updating all extensions and resetting my venv (deleting it and running webui.bat). 

So I guess it is compatible with Adetailer now? The errors I saw before aren't popping up.
Pinging @v0xie to let you know of this, I honestly don't know why it works on my end now.
